package-lock.json

```diff
@@ -5,11 +5,12 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "npm7-test",
```

Comment:
Was surprised by this but seems like npm 7 adds this when running `npm install lodash@v`, not sure why it didn't get added from the start but seems like an npm issue.